patches/third_party-blink-renderer-modules-plugins-dom_plugin_array.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
+index 322e74880ddaa6c6c54f80f4f5f7ab0264631ffd..0c13152cb2f9ebd5bbf86955f50eab6909f05443 100644
+--- a/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
++++ b/third_party/blink/renderer/modules/plugins/dom_plugin_array.cc
+@@ -146,6 +146,7 @@ void DOMPluginArray::UpdatePluginData() {
+       }
+     }
+   }
++  BRAVE_DOM_PLUGINS_UPDATE_PLUGIN_DATA
```

Comment:
I'm ok with this patch for now, but I think this could be done in the browser process without patching and if we have similar things in the future I think we should do a little refactoring so we can easily access the domain key (without duplicating logic) there